components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -1065,7 +1065,14 @@ $xwiki.ssx.use($serviceDocName)
   #elseif ($ownerReference.equals($xcontext.userReference))
     #set ($ownerName = 'you')
   #else
-    #set ($ownerName = $xwiki.getUserName("$ownerReference", false))
+    #if($owner.isGroup())
+      #set ($ownerName = $xwiki.getDocument($ownerReference.toString()).getTitle())
+      #if ("$!ownerName"=='')
```

Comment:
Actually you should use `#set ($ownerName = $xwiki.getDocument($ownerReference).plainTitle)`, this one already falls back to the document name if a title is not set, plus it is safer since it discards any wiki or HTML markup, which is safer.